test/e2e/e2eutil/util.go

```diff
@@ -27,9 +27,9 @@ import (
 	"k8s.io/client-go/pkg/api/v1"
 )
 
-func KillMembers(kubecli kubernetes.Interface, ns string, names ...string) error {
+func KillMembers(kubeClient kubernetes.Interface, namespace string, names ...string) error {
```

Comment:
`e2eutil/wait_util.go` and `e2eutil/tpr_util.go` all use the name `kubeClient`. Should I change those to kubeCli too?